task/backend/executor/executor_metrics.go

```diff
@@ -83,6 +84,13 @@ func NewExecutorMetrics(te *TaskExecutor) *ExecutorMetrics {
 			Name:      "resume_runs_counter",
 			Help:      "Total number of runs resumed by task ID",
 		}, []string{"taskID"}),
+
+		scheduleInterval: prometheus.NewHistogramVec(prometheus.HistogramOpts{
+			Namespace: namespace,
+			Subsystem: subsystem,
+			Name:      "schedule_interval",
+			Help:      "The interval between the time the run was scheduled for and the time the task's next run is due at, by task type",
```

Comment:
The name and wording of this metric is a little confusing. I would consider calling the field `runLatency` and the metric `run_latency_seconds`. [Prometheus guidelines](https://prometheus.io/docs/practices/naming/#metric-names) suggest adding the units (plural) to the end of the metric name.

The `Help` could the be something like:

> Records the latency between the time a task was due to run and the time the task started execution, by task type